sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,162 @@
+
+```python
+search_result = client.search_address("400 Broad, Seattle");
+```
+
+### Search for an address or Point of Interest
+
+You can use Fuzzy Search to search an address or a point of interest (POI). The following examples demostrate how to search for `pizza` over the scope of a specific country (`France`, in this example).
+
+```python
+fuzzy_search_result = client.fuzzy_search({ query: "pizza", country_filter: "fr" });
+```
+
+### Make a Reverse Address Search to translate coordinate location to street address
+
+You can translate coordinates into human readable street addresses. This process is also called reverse geocoding.
+This is often used for applications that consume GPS feeds and want to discover addresses at specific coordinate points.
+
+```python
+coordinates: LatLon = {
+  latitude: 47.59118,
+  longitude: -122.3327,
+};
+
+reverse_search_result = client.reverse_search_address(coordinates);
+```
+
+### Translate coordinate location into a human understandable cross street
+
+Translate coordinate location into a human understandable cross street by using Search Address Reverse Cross Street API. Most often, this is needed in tracking applications that receive a GPS feed from a device or asset, and wish to know where the coordinate is located.
+
+```python
+coordinates: LatLon = {
+  latitude: 47.59118,
+  longitude: -122.3327,
+};
+
+const reverse_search_result = client.reverse_search_cross_street_address(coordinates);
+```
+
+### Sample Code
+
+These are code samples that show common scenario operations with the Azure Maps Search client library.
+The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations.
+Before run the sample code, refer to Prerequisites
+<!-- [Prerequisites](#Prerequisites) -->
+to create a resource, then set some Environment Variables
+
+```bash
+set AZURE_MAPS_SERVICE_ENDPOINT="https://<RESOURCE_NAME>.mapsservices.azure.com"
+set MAPS_SAMPLES_CONNECTION_STRING="<connection string of your Maps service>"
+
+pip install azure-maps-search
+
+python samples\search.py
+```
+
+## Troubleshooting
+
+Running into issues? This section should contain details as to what to do there.
+
+### Next steps
+
+More sample code should go [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-search/samples), along with links out to the appropriate example tests.
+
+### Contributing
+
+If you encounter any bugs or have suggestions, please file an issue in the [Issues](<https://github.com/Azure/azure-sdk-for-python/issues>) section of the project.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fsdk%2Ftemplate%2Fazure-template%2FREADME.png)
```

Comment:
What's this link?
https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fsdk%2Ftemplate%2Fazure-template%2FREADME.png

Reply:
All the SDK had this line at the end, I assume this is for extra `.` or line at the end of README.